app/index.tsx

```diff
@@ -0,0 +1,21 @@
+'use client';
```

Comment:
I could imagine putting this stuff in layout.tsx, so it would at least handle children of this route if we make any.

I suppose the safest thing to do might be to use middleware to require it for everything in the app.

I'm not sure any of these things need to be done at all, let alone now, I'm just trying to get my head around the options/tradeoffs here, so I'd love your thoughts.

Reply:
Yep when I finally did get it, I realized it wasn't routing through `Index()` at all. This whole file should be removed now!